### PR TITLE
update deps & switch to github actions

### PR DESCRIPTION
**Pull request by aotarola**
---
_This PR was started by: [git wf pr](https://github.com/groupon/git-workflow/releases/tag/v1.3.1)_

### CODE REVIEW

**Comment by dbushong** [comment]
This PR doesn't seem to update chromedriver(?)

**Comment by aotarola** [comment]
Yeah, I got inspired to switch to gh actions, but forgot to update the title, fixed now 😅

**Comment by aotarola** [comment]
@dbushong got a green now, please review 👀

File: lib/browser/lighthouse.js [hunk 1]
```diff
@@ -131,18 +131,16 @@ function parseLhResult(results) {
...
      */
     const auditInfo = { description, details };
+    const snippets = [];
 
     if (score) {
       successObj[id] = auditInfo;
     } else if (scoreDisplayMode === 'notApplicable') {
       notApplicableObj[id] = auditInfo;
     } else if (scoreDisplayMode !== 'manual') {
-      const snippets = [];
```
Comment (dbushong) [inline lib/browser/lighthouse.js:140]: **nit (non-blocking):** Isn't `snippets` only used inside this else/if clause?  Why not leave the declaration in here?

**Review by dbushong — APPROVED** [review]
LGTM.  What was the reason for the draggable -> pwa switch?  (And score change?)

File: lib/browser/lighthouse.js (hunk 1 above)
Comment (aotarola) [reply lib/browser/lighthouse.js:140]: a bad refactoring from my side, fixed!

**Comment by aotarola** [comment]
@dbushong, since I updated lighthouse dep; the scoring for PWA, changed as well, so I created a new PWA page test in https://github.com/testiumjs/testium-example-app/pull/21, it should cover it properly now.

**Pull request merged**